components/combobox/combobox_empty-list.vue

```diff
@@ -4,9 +4,11 @@
     data-qa="dt-combobox-empty-list"
   >
     <dt-list-item
-      class="d-d-flex d-jc-center"
+      class="dt-empty-list-item d-fs-200 d-lh4 d-py4 d-px12 d-d-flex d-jc-center d-ai-center"
```

Comment:
with so many default utility classes in here i'd worry that some of them would not be overridden by `itemclass` due to `!important`

Reply:
yeah I mean if you use other utility classes will override due to the order, but using custom classes you will need to use `!important` in the already defined default classes. So moved the default styles to a class. https://github.com/dialpad/dialtone-vue/pull/601/commits/e65f25683e0256e9fa1a7ad52e723e0042312057